.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+  schedule:
```

Comment:
Why do we want to enable the tests only on schedule? Is there any condition that prevents us from running tests on each PR and push to master?

Reply:
The tests are exectued already on push to master. I removed the branch from pull_request, so it should run for all PRs